.github/workflows/add-to-project.yaml

```diff
@@ -6,7 +6,7 @@ on:
       - opened
       - reopened
       - transferred
-  pull_request:
+  pull_request_target:
```

Comment:
I've heard a fair bit about secret exfiltration from repositories using this feature, but I don't know the details. Does allowing forks to access our secrets in this way potentially expose those secrets to code in the fork? Are the tokens here allowed to do anything sensitive?

Reply:
I discussed it with @rubensf who confirmed those issues don't apply in this case, and this is safe to do.

Reply:
See also the TL;DR from [here](https://securitylab.github.com/research/github-actions-preventing-pwn-requests/):

> Combining `pull_request_target` workflow trigger with an explicit checkout of an untrusted PR is a dangerous practice that may lead to repository compromise.

We aren't doing any checkout here, and are using `pull_request_target` for its intended purpose: to process PRs safely. Also from that page:

> `pull_request_target` runs in the context of the target repository of the PR, rather than in the merge commit. This means the standard checkout action uses the target repository to prevent accidental usage of the user supplied code.